Mopups/Mopups.Maui/Platforms/Android/Impl/AndroidMopups.cs

```diff
@@ -31,8 +31,6 @@ public static bool SendBackPressed(Action? backPressedHandler = null)
             return true;
         }
 
-        backPressedHandler?.Invoke();
```

Comment:
And this is why i made a PR ahaha.

So as far as im aware, if i keep that invoke in it makes an endless loop which crashes the system. 

I think the original intention is to provide extra functionality on top.

The apphostbuilder area seems to be the only place this is registered, so perhaps we instead need to move that custom functionality to there somehow?

I'm on mobile, but i'll give this another shakedown, i need to learn more about apphostbuilder instead of just throwing everything at it and seeing what sticks